src/odemis/driver/xt_client.py

```diff
@@ -1447,17 +1455,41 @@ def _cancelAutoContrastBrightness(self, future):
     #         return True
 
 
+class FibScanner(model.Emitter):
+    """
+    This is an extension of the model.Emitter class for controlling the FIB (focused ion beam). Currently via XTLib
+    only minimal control of the FIB is possible.
+    """
+
+    def __init__(self, name, role, parent, channel="ion2", **kwargs):
+        """
+
+        :param name (str):
+        :param role (str):
+        :param parent (SEM object):
+        :param channel (str): Specifies the type of scanner (alphabetic part) and the quadrant it is displayed in (
+        nummerical part) on which the scanning feed is displayed on the Microscope PC in the Microscope control window
+        of TFS. The quadrants are numbered 1 (top left) - 4 (right bottom).
+        ion2 is set as default since the top left quadrant is used as default for the ebeam (electron1)
```

Comment:
"top left quadrant is used as default for the ebeam (electron1)" ->  "top right quadrant is used as default for the FIB image" (or adjust to whatever is the TFS software behaviour)